static/app/views/dashboardsV2/controls.tsx

```diff
@@ -90,18 +100,41 @@ class Controls extends React.Component<Props> {
       <StyledButtonBar gap={1} key="controls">
         <DashboardEditFeature>
           {hasFeature => (
-            <Button
-              data-test-id="dashboard-edit"
-              onClick={e => {
-                e.preventDefault();
-                onEdit();
-              }}
-              priority="primary"
-              icon={<IconEdit size="xs" />}
-              disabled={!hasFeature}
-            >
-              {t('Edit Dashboard')}
-            </Button>
+            <React.Fragment>
+              <Button
+                data-test-id="dashboard-edit"
+                onClick={e => {
+                  e.preventDefault();
+                  onEdit();
+                }}
+                priority="primary"
+                icon={<IconEdit size="xs" />}
+                disabled={!hasFeature}
+              >
+                {t('Edit Dashboard')}
+              </Button>
```

Comment:
you might have some merge conflicts with some of the widget library changes here!